tabula/io.py

```diff
@@ -678,10 +678,19 @@ def _extract_from(raw_json, pandas_options=None):
 
         df = pd.DataFrame(data=list_data, columns=_columns, **pandas_options)
 
-        for c in df.columns:
-            df[c] = pd.to_numeric(df[c], errors="ignore")
+        if pandas_options.get('dtype', None) == int:
+            for c in df.columns:
+                df[c] = pd.to_numeric(df[c], errors="ignore", downcast='integer')       
+        elif pandas_options.get('dtype', None) == str:
+            mask = df.applymap(lambda x: x == 'None')
+            for col in df.columns[(mask).any()]:
+                df.loc[mask[col], col] = None
+        else:
+            for c in df.columns:
+                df[c] = pd.to_numeric(df[c], errors="ignore")
```

Comment:
I mean, this would be enough for your motivation. Could you try it with your data?

```suggestion
        if not pandas_options.get('dtype'):
            for c in df.columns:
                df[c] = pd.to_numeric(df[c], errors="ignore")
```

Reply:
Its works in my data with your suggestion :100: